### DISCUSSION TITLE
Incompatible configuration not working

### DISCUSSION THREAD

**Discussion by alekfestekjian** [nftchef/art-engine#184] [Help + Q&A]
I'm trying to make it so that the layers in "8_Skins_ArmsDown" doesn't match with any of the layers in the Bat, Gloves and Superheros directory. The reason I have 8_Skin_ArmsDown#100 is just for testing because essentially each skin sub-directory can only match with layers from one of the sub-directories in "Outfits". I know I could do multiple layer configurations but I am trying to get it done with incompatible layers. Can anyone provide some guidance? Currently when I run this it still matches layers from Bat, Gloves and superheros with the "8_Skins_ArmsDown" layer.

<img width="957" alt="Screen Shot 2022-05-20 at 1 09 04 PM" src="https://user-images.githubusercontent.com/6645660/169578452-28b300ac-023e-4cb4-a9b7-6fb43cd86f3d.png">

**Comment by jjclay** [comment]
The 8_Skin_ArmsDown layer must come before the Bat, Gloves and superheros layers in order to make them incompatible.  You can use the z-indexing feature in order to rearrange them.  https://generator.nftchef.dev/readme/z-index-layer-order

**Comment by CarrouselNFT** [comment]
Hey look here #150
